plugin/ctrlsf.vim

```diff
@@ -58,16 +58,16 @@ endf
 " current visual selection normally, so I need to workaround it.
 func! s:SearchVwordCmd(type, to_exec)
     let keys = '":\<C-U>'. a:type .' " . g:CtrlSFGetVisualSelection()'
-    let keys .= a:to_exec ? '."\r"' : '." "'
-    let cmd = ":\<C-U>call feedkeys(" . keys . ", 'n')\r"
+    let keys .= a:to_exec ? '."\<CR>"' : '." "'
+    let cmd = ":\<C-U>call feedkeys(" . keys . ", 'n')\<CR>"
     return cmd
 endf
 " }}}
 
 " s:SearchPwordCmd() {{{2
 func! s:SearchPwordCmd(type, to_exec)
     let cmd = ":\<C-U>" . a:type . " " . @/
-    let cmd .= a:to_exec ? "\r" : " "
+    let cmd .= a:to_exec ? "\<CR>" : " "
```

Comment:
I know ,just make the code more readable ,as we know ,most people use `<CR>` for `Enter` command.maybe it is the first time I see someone use `\r` for `Enter`.